<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditionElements</class>
 <widget class="QDialog" name="EditionElements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="np_l">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Узел n+</string>
   </property>
  </widget>
  <widget class="QLabel" name="p5">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>310</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ИТУТ</string>
   </property>
  </widget>
  <widget class="QLabel" name="nm_l">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Узел n-</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="node_nm">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>190</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QLabel" name="p3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>270</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ИТУТ</string>
   </property>
  </widget>
  <widget class="QLabel" name="p1">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>230</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ИТУТ</string>
   </property>
  </widget>
  <widget class="QLabel" name="km_l">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>190</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Узел k-</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="node_kp">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>150</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QLabel" name="p2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сопротивление (КОм)</string>
   </property>
  </widget>
  <widget class="QLabel" name="p0">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сопротивление (КОм)</string>
   </property>
  </widget>
  <widget class="QLabel" name="p4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сопротивление (КОм)</string>
   </property>
  </widget>
  <widget class="QLabel" name="kp_l">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>150</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Узел k+</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p2_value">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QSpinBox" name="node_np">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p0_value">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>230</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QSpinBox" name="node_km">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>190</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p4_value">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>310</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="change">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Внести изменения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Список компонентов</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Номер элемента</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="elenent_number">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>360</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Возврат к основному окну</string>
   </property>
  </widget>
  <widget class="QListWidget" name="component_list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>256</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p3_value">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>270</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p5_value">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>310</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p1_value">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>230</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
